xtensorConfig.cmake.in

```diff
@@ -35,32 +35,45 @@ if(XTENSOR_USE_TBB)
     target_compile_definitions(@PROJECT_NAME@ INTERFACE XTENSOR_USE_TBB)
 endif()
 
-if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} VERSION_GREATER_EQUAL 3.11)
```

Comment:
Aren't you removing this version check here? I think that was a big reason why people had issues.

Reply:
We introduced the version check *because* people had problems. As far as I understand now it seems that the commands like `target_compile_options` can only act on `IMPORTED` targets since recently. However, setting the target's variables, using `set_property(TARGET ... PROPERTY ... ...)` is allowed in all versions. So this replacement should render the extra targets completely robust against CMake versions. 

Now, there is a question how to know for sure. I tested against linux/trusty on Travis (https://github.com/tdegeus/test_cmake) whereby the current implementation failed but the new implementation passed, but that is of course not a guarantee. That is good to have some confirmations from those who had problems before.